Source/ThirdParty/libwebrtc/Source/webrtc/modules/congestion_controller/goog_cc/loss_based_bwe_v2.cc

```diff
@@ -508,7 +508,7 @@ absl::optional<LossBasedBweV2::Config> LossBasedBweV2::CreateConfig(
   if (!enabled.Get()) {
     return config;
   }
-  config.emplace();
+  config.emplace(Config());
```

Comment:
This change is needed to fix the build.

Firefox implements the same fix:
https://github.com/ErichDonGubler/firefox/commit/cf442c0b6657198dbf8d0ea925eebf6be7d453be.